docs/intro/tutorial08.txt

```diff
@@ -48,12 +48,15 @@ The steps are not duplicated in this tutorial, because as a third-party
 package, it may change separately to Django's schedule.
 
 Once installed, you should be able to see the DjDT "handle" on the right side
-of the browser window when you refresh the polls application. Click it to open
-the debug toolbar and use the tools in each panel. See the `panels
-documentation page
-<https://django-debug-toolbar.readthedocs.io/en/latest/panels.html>`__ for more
+of the browser window when you browse to the admin of your project (http://localhost:8000/admin). 
+Click it to open the debug toolbar and use the tools in each panel. 
+See the `panels documentation page <https://django-debug-toolbar.readthedocs.io/en/latest/panels.html>`__ for more
 information on what the panels show.
 
+To see the toolbar in the polls application make sure all your templates
+are using `complete HTML documents <https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#anatomy_of_an_html_document>`_.
+The toolbar will only render in responses that contain the ``<body></body>`` tags.
+
```

Comment:
```suggestion
of the browser window when you browse to ``http://localhost:8000/admin/``.
Click it to open the debug toolbar and use the tools in each panel. See the
`panels documentation page`__ for more information on what the panels show.

To see the DjDT "handle" in the polls application make sure all your templates
are using `complete HTML documents`__. The "handle" will only render in
responses that contain ``<body></body>`` tags.

__ https://django-debug-toolbar.readthedocs.io/en/latest/panels.html
__ https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#anatomy_of_an_html_document

```

- Applied some [line length wrapping](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files)
- Previously we call it the `DjDT "handle"` and then `the toolbar` (tweaked)

Reply:
Thanks @sarahboyce. Did you use one of the commands in the `makefile` to automatically format these or is it done manually? Thanks